pkg/network/setup/netconf.go

```diff
@@ -77,6 +77,11 @@ func NewNetConfWithCustomFactoryAndConfigState(nsFactory nsFactory, cacheCreator
 
 // Setup applies (privilege) network related changes for an existing virt-launcher pod.
 func (c *NetConf) Setup(vmi *v1.VirtualMachineInstance, networks []v1.Network, launcherPid int, preSetup func() error) error {
+	podIfaceNamesByNetworkName, err := vmispec.IndexPodIfaceNamesByNetworkName(vmi.Spec.Networks, vmi.Status.Interfaces)
+	if err != nil {
+		return fmt.Errorf("missing pod interface naming info: %v", err)
```

Comment:
nit: Please add the `vmi.Status.Interfaces` list to the error.

Reply:
This change was reverted.